rllib/env/multi_agent_env.py

```diff
@@ -702,18 +742,27 @@ def __init__(self, env: MultiAgentEnv, return_error_as_obs: bool = False):
         self.last_obs = {}
         self.last_rewards = {}
         self.last_dones = {"__all__": False}
+        self.last_truncateds = {}
         self.last_infos = {}
 
     def poll(
         self,
-    ) -> Tuple[MultiAgentDict, MultiAgentDict, MultiAgentDict, MultiAgentDict]:
+    ) -> Tuple[
+        MultiAgentDict,
+        MultiAgentDict,
+        MultiAgentDict,
+        MultiAgentDict,
+        MultiAgentDict,
+    ]:
         if not self.initialized:
+            # TODO(sven): Should we make it possible to pass in a seed here?
```

Comment:
No, not necessary. Users should call reset themselves for that

Reply:
But they can't. This logic is normally entirely encapsulated in our RolloutWorker/Sampler/EnvRunner logic, which starts the endless loop right away with a `poll()` call (NOT a reset).